index.html

```diff
@@ -160,6 +160,61 @@ <h2 class="media-heading  text-center" style="color:black;">Mentorship</h2>
     </div>
   </section>
 
+
+<!-- Testimonials section -->
+<section id="testimonials">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h2 class="section-title wow fadeInUp" data-wow-delay="0.3s">Testimonials</h2>
+      </div>
+      <div class="col-md-4">
+        <div class="testimonial-box wow fadeInUp" data-wow-delay="0.4s">
+          <div class="testimonial-author">
+            <img src="images/testi_1.jpg" class="img-circle" alt="Testimonial Author">
+            <h4>John Doe</h4>
+            <p>CEO, ABC Company</p>

```

Comment:
Could you please remove that ABC or XYZ or something company name,
it will prefer to not to have a company name either

Reply:
I have removed it.Please accept